src/benchmarklib/benchmark_runner.cpp

```diff
@@ -51,6 +51,12 @@ BenchmarkRunner::BenchmarkRunner(const BenchmarkConfig& config, std::unique_ptr<
 
     const auto scheduler = std::make_shared<NodeQueueScheduler>();
     CurrentScheduler::set(scheduler);
+
+    if (config.clients == 1) {
```

Comment:
I'd put this into BenchmarkRunner::run() so that it prints when I - as a creator of the BenchmarkRunner - would probably more expect it to print.

Reply:
cli_config_parser.cpp is probably even better